src/cluecode/finder_data.py

```diff
@@ -200,18 +217,24 @@ def set_from_text(text):
    .png
    .jpg
    .gif
+   .jpeg
 ''')))
 
 
-def classify(s, data_set, suffixes=None):
+def classify(s, data_set, suffixes=None, ignored_hosts=None):
     """
     Return True or some classification string value that evaluates to True if
     the data in string s is not junk. Return False if the data in string s is
-    classified as 'junk' or uninteresting.
+    classified as 'junk' or uninteresting. Return False is domain is in `ignored_hosts`.
     """
     if not s:
         return False
     s = s.lower().strip('/')
+    # Separate test for emails - need to ignore xyz@some.com, but not say, xyz@gruesome.com
+    if '@' in s and ignored_hosts:
+        host_name = s.rpartition('@')[-1]
```

Comment:
Rather than using [-1] use something like this that is a tad more explicit:
`_name, _at, host_name = s.rpartition('@')`

Reply:
Done, thank you!